mautic/mautic
# Fix "Send a webhook" action does not work in campaigns

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


Step 1: Run Mautic
Step 2: Create a campaign with "Send a webhook" action(with proper segment, contacts etc.). Campaign can contain just "Send a webhook" action, add some data fields with POST request
Step 3: Trigger campaign, see that "Send a webhook" is successfully completed in campaign Action summary.
Step 4: Check details of one of contacts that was in the campaign, you will see that webhook action failed;




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10580"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
